modules/nested-elements/assets/js/editor/nested-tabs/hooks/data/document/repeater/insert/nested-tabs-adjust-new-container-title.js

```diff
@@ -0,0 +1,51 @@
+import Base from 'elementor/modules/nested-elements/assets/js/editor/hooks/data/document/repeater/base';
+
+export class NestedTabsAdjustNewContainerTitle extends Base {
```

Comment:
Explain the difference from `nested-tabs-adjust-container-titles.js`

Reply:
Handled.